setup.py

```diff
@@ -3,7 +3,7 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
```

Comment:
There is a potential that ```setuptools``` might not be available on a system. I would suggest putting in this wrapper to allow fallback:

```
try:
  from setuptools import setup
except ImportError:
  from distutils.core import setup
```